src/components/sbb-link/sbb-link.tsx

```diff
@@ -46,14 +46,20 @@ export class SbbLink {
    */
   @Prop() public iconFlip?: boolean;
 
+  /**
+   * If this is set to true an span element will be used
+   * instead of a anchor or a button
+   */
+  @Prop() public staticSpan?: boolean = false;
```

Comment:
if it is initialised per default ("=false"),  no need of the "?". It is anyway optional,because it has a default initalizer.